blocks/editable/index.js

```diff
@@ -194,10 +194,41 @@ export default class Editable extends wp.element.Component {
 	}
 
 	onKeyDown( event ) {
+		const { keyCode } = event;
+		const moveUp = ( keyCode === UP || keyCode === LEFT ) && this.isStartOfEditor();
+		const moveDown = ( keyCode === DOWN || keyCode === RIGHT ) && this.isEndOfEditor();
+		const selectors = [
+			'*[contenteditable="true"]',
+			'*[tabindex]',
```

Comment:
Wow, proper tabbable detection is complex!

https://github.com/medialize/ally.js/blob/master/src/is/tabbable.js